src/pages/SingleTask/SingleTask.jsx

```diff
@@ -0,0 +1,218 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import { StopWatch } from "../../components/StopWatch";
+import { useTasks } from "../../context/task-context";
+import { findTask } from "../../utils/findTask";
+import NotPinned from "../../assets/notPinned.png";
+import NotPinnedDark from "../../assets/notPinnedDark.png";
+import Archive from "../../assets/archive.png";
+import ArchiveDark from "../../assets/archiveDark.png";
+import Unarchive from "../../assets/unarchive.png";
+import UnarchiveDark from "../../assets/unarchiveDark.png";
+import Restore from "../../assets/restore.png";
```

Comment:
Just a suggestion: You can have a look at absolute imports, they make life easier, especially in big projects